pkgs/development/python-modules/dbt/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, fetchPypi, buildPythonPackage, autopep8 }:
+
+buildPythonPackage rec {
+  pname = "dbt";
+  version = "0.19.0";
+
+  propagatedBuildInputs = [ autopep8 ];
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "276ced7e9e3cb22e5d7c14748384a5cf5d9002257c0ed50c0e075b68011bb6d0";
+  };
+
+  meta = with lib; {
+    description = ''
+      dbt (data build tool) is a command line tool that helps analysts and
+      engineers transform data in their warehouse more effectively
+    '';
+    homepage = "https://pypi.python.org/pypi/dbt/";
```

Comment:
I would suggest to actually point to upstream's repo (https://github.com/dbt-labs/dbt-core) or the documentation (https://docs.getdbt.com/docs/).